sysutils/mackup/Portfile

```diff
@@ -22,11 +22,12 @@ long_description    Mackup backs ups your application settings in a safe \
 homepage            https://github.com/lra/mackup
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 
-checksums           rmd160  fab8f379f158988c21bc761a57848cda46a9b149 \
-                    sha256  4f3886cc5e0bf645c4dded8e3fb0d26076d4c7908dd1f6586f6dabac08e0b6bd \
-                    size    44262
+checksums           rmd160  9b3575b3eb82a9bea117ae135109be6c628c3c00 \
+                    sha256  6918d9caba1c0e849f63f1868ce3c51e87d33ce0e5a5eb4266a553b6ac22871e \
+                    size    53412
 
-python.default_version 36
+python.versions         36 38
```

Comment:
Since this is not a python module and it is only using a single python, the `python.versions` line should be removed.

Reply:
Thanks @mf2k. I've removed the `python.versions` line, but left the `python.default_version` at 38.